File: lang/csharp/src/MessageException.cs
namespace muscle.message {

  /// <summary>
  /// Base class for the various Exceptions that the Message class may throw.
  /// </summary>
  public class MessageException : System.Exception {
    public MessageException(string s) : base(s) { }
      
    public MessageException() : 
      base("Error accessing a Message field") { }
  }
}

File: lang/java_j2me/src/com/lcs/micromuscle/message/MessageException.java
/* This file is Copyright 2001 Level Control Systems.  See the included LICENSE.TXT file for details. */
package com.meyer.micromuscle.message;

/** Base class for the various Exceptions that the Message class may throw */
public class MessageException extends Exception
{
   private static final long serialVersionUID = -7107595031653595454L;

   public MessageException(String s)
   {
      super(s);
   }
    
   public MessageException()
   {
      super("Error accessing a Message field");
   }
}
